modules/default/newsfeed/newsfeed.js

```diff
@@ -8,13 +8,18 @@
  */
 
 Module.register("newsfeed",{
-
 	// Default module config.
 	defaults: {
 		feeds: [
 			{
 				title: "New York Times",
 				url: "http://www.nytimes.com/services/xml/rss/nyt/HomePage.xml",
+				className: "nyt",
```

Comment:
You can set this in the example configuration file, not here because this affect another instalations.